### PR TITLE
doc: bluetooth: Add existing qualification listings

### PR DESCRIPTION
**Pull request by carlescufi**
List the currently available Bluetooth Qualification Listings for the
Zephyr Host and Controller.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

**Review by nashif — APPROVED** [review]

File: doc/subsystems/bluetooth/qualification.rst [hunk 1]
```diff
@@ -15,3 +15,28 @@ the following documents:
...
+**********************
+
+The Zephyr BLE stack has obtained qualification listings for both the Host
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
```
Comment (dbkinder) [inline doc/subsystems/bluetooth/qualification.rst]: While this table works, you'll find using a list-table directive would be much easier to maintain:

    .. list-table::
       :header-rows: 1

       * - Zephyr version
         - Link
         - Qualifying Company

       * - 1.13
         - `Launch Studio details <https://launchstudio.bluetooth.com/ListingDetails/70189>`__
         - Nordic

File: doc/subsystems/bluetooth/qualification.rst [hunk 2]
```diff
@@ -15,3 +15,28 @@ the following documents:
...
+Zephyr version  Link                                                    Qualifying Company
+=============== ======================================================= ===================
+1.13            https://launchstudio.bluetooth.com/ListingDetails/70189 Nordic
+=============== ======================================================= ===================
+
+Controller qualifications
+-------------------------
+
+=============== ======================================================= =================== ===================
```
Comment (dbkinder) [inline doc/subsystems/bluetooth/qualification.rst]: Same here:

    .. list-table::
       :header-rows: 1

       * - Zephyr version
         - Link
         - Qualifying Company
         - Compatible Hardware

       * - 1.9 to 1.13
         - `Launch Studio details <https://launchstudio.bluetooth.com/ListingDetails/25166>`__
         - Nordic
         - nRF52x

File: doc/subsystems/bluetooth/qualification.rst [hunk 3]
```diff
@@ -15,3 +15,28 @@ the following documents:
...
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
+Zephyr version  Link                                                    Qualifying Company
+=============== ======================================================= ===================
+1.13            https://launchstudio.bluetooth.com/ListingDetails/70189 Nordic
```
Comment (dbkinder) [inline doc/subsystems/bluetooth/qualification.rst]: These links to launchstudio require a login/account to see any details :(

File: doc/subsystems/bluetooth/qualification.rst [hunk 4]
```diff
@@ -15,3 +15,28 @@ the following documents:
...
+=============== ======================================================= ===================
+
+Controller qualifications
+-------------------------
+
+=============== ======================================================= =================== ===================
+Zephyr version  Link                                                    Qualifying Company  Compatible Hardware
+=============== ======================================================= =================== ===================
+1.9 to 1.13     https://launchstudio.bluetooth.com/ListingDetails/25166 Nordic              nRF52x
```
Comment (dbkinder) [inline doc/subsystems/bluetooth/qualification.rst]: The launchstudio page (that I can see without a login/account) only mentions Zephyr version 1.9x

**Review by dbkinder — DISMISSED** [review]
For maintainability, recommend using a list-table directive

**Review by jhedberg — APPROVED** [review]

File: doc/subsystems/bluetooth/qualification.rst (hunk 2 above)
Comment (carlescufi) [reply doc/subsystems/bluetooth/qualification.rst]: will switch, thanks!

File: doc/subsystems/bluetooth/qualification.rst (hunk 3 above)
Comment (carlescufi) [reply doc/subsystems/bluetooth/qualification.rst]: They do? that is strange. OK I will figure it out

File: doc/subsystems/bluetooth/qualification.rst (hunk 4 above)
Comment (carlescufi) [reply doc/subsystems/bluetooth/qualification.rst]: again strange, no idea why that is the case, i will check with someone

File: doc/subsystems/bluetooth/qualification.rst (hunk 3 above)
Comment (dbkinder) [reply doc/subsystems/bluetooth/qualification.rst]: I can see the page you're linking to, but if I click on the details link at the bottom of the page, it requires me to login:

image

File: doc/subsystems/bluetooth/qualification.rst (hunk 4 above)
Comment (dbkinder) [reply doc/subsystems/bluetooth/qualification.rst]: Here's what I see...

image

File: doc/subsystems/bluetooth/qualification.rst (hunk 3 above)
Comment (nashif) [reply doc/subsystems/bluetooth/qualification.rst]: why is this a concern? It is not on a Zephyr page. The landing page is the important one.

**Review by joerchan — APPROVED** [review]

**Review by sjanc — APPROVED** [review]

**Comment by carlescufi** [comment]
@dbkinder ping

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Pull request merged**